cl/alerts/management/commands/cl_send_alerts.py

```diff
@@ -232,13 +206,9 @@ def run_query(self, alert, rate, v1_webhook=False):
             qd["filed_after"] = cut_off_date
         elif query_type == SEARCH_TYPES.ORAL_ARGUMENT:
             qd["argued_after"] = cut_off_date
-            if waffle.switch_is_active("oa-es-alerts-active"):
-                # Return empty results for OA alerts. They are now handled
-                # by Elasticsearch.
-                return query_type, results, v1_results
```

Comment:
Shouldn't we keep the `return` statement? It appears that the previous behavior, when`oa-es-alerts-active` was active, was to simply return. This PR, however, removes that return statement.

Reply:
Yeah! I got confused here, and this was a mistake. Thanks for catching it. I've refactored the command so it now only works for `SEARCH_TYPES.OPINION` alerts.